sklearn/neighbors/_binary_tree.pxi

```diff
@@ -1620,11 +1620,12 @@ cdef class BinaryTree:
             - 'linear'
             - 'cosine'
             Default is kernel = 'gaussian'
-        atol, rtol : float, default=0, 1e-8
+        atol : float, default=0
```

Comment:
To resolve `test_docstring_parameters`. Building with the newest version of Cython now exposes these docstrings correctly.